packages/tabmanager-extension/src/index.ts

```diff
@@ -64,6 +64,13 @@ const plugin: JupyterFrontEndPlugin<void> = {
         labShell.layoutModified.connect(() => {
           populate();
         });
+
+        // when current tab changes, make sure it is activated
```

Comment:
My inclination is that this logic should be in application-extension, since it is foundational and someone could disable this extension.

Reply:
sure, will do 👍